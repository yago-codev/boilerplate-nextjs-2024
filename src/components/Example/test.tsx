import { render, screen } from '@testing-library/react'

import Example from '.'

describe('<Example />', () => {
  it('should render the heading', () => {
    const { container } = render(<Example />)

    expect(screen.getByText(/Example/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
