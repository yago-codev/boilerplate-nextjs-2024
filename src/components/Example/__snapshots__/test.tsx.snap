// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Example
  </h1>
</main>
`;
